app/views/handlers/list_add_question.py

```diff
@@ -13,4 +13,8 @@ def handle_post(self):
         )
         self.questionnaire_store_updater.update_answers(self.form.data, list_item_id)
 
+        self.questionnaire_store_updater.update_same_name_items(
+            self.parent_block["for_list"], self._get_same_name_answer_ids()
+        )
+
```

Comment:
Could the updating of the same name items be done in the `handle_post` of `list_action.py` rather than in each of the add/edit/remove actions?